rpc_jobs/multi_node_aio.yml

```diff
@@ -49,9 +157,9 @@
         stage('OnMetal Multi-Node AIO') {{
           git branch: "{branch}", url: "https://github.com/rcbops/rpc-openstack"
           if("{series}" == "mitaka"){{
-            build job: 'OnMetal-Multi-Node-AIO_mitaka-trusty-post-merge'
+            build job: 'OnMetal-Multi-Node-AIO_mitaka-trusty-deploy-post-merge'
```

Comment:
I think these jobs are outside the scope of this work. I only changed what was required to allow them to continue to work. However, looking at them again I forgot to add the resource bit to the name so will update that.